docs/UserGuide.md

```diff
@@ -442,13 +465,16 @@ Arguments similar to `delete_session` command except the following:
 </div>
 
 Example:
-* John Doe has a recurring session from 2021-03-17 to 2021-04-14.
-* `delete_rec_session n/John Doe i/1 d/2021-03-31` deletes a valid single session dated 2021-03-31
-from an existing recurring session. 
-* This will split the recurring session into two recurring sessions that span the period exclusively
-before and after 2021-03-31 as shown below.
+1. You would like to delete an existing session from a recurring session for **John Doe** on the date **2021-01-16** at **18:00**.
+2. Locate the recurring session index to delete on the right of the session card.
+
+![delete_recurring_session_example](images/choonwei/delete_recurring_session.png)
+
+3. Using the index from step 2, type in `delete_rec_session n/John Doe i/1 d/2021-01-16` which deletes a single session from **John Doe**'s recurring session.
+4. You will now see that this splits the original recurring session into two recurring session from **2021-01-02** to **2021-01-09** and
+   another from **2021-01-23** to **2021-01-30**.
```

Comment:
should we follow the date format as displayed in the session card? i.e. 02 Jan 2021

Reply:
Agreed! Made changes!